cmd/world/root/root.go

```diff
@@ -44,6 +52,63 @@ func init() {
 	logger.AddLogFlag(doctorCmd)
 }
 
+func checkLatestVersion() error {
+	url := "https://api.github.com/repos/Argus-Labs/world-cli/releases/latest"
+
+	// Create a request with the context
+	req, err := http.NewRequestWithContext(context.Background(), http.MethodGet, url, nil)
+	if err != nil {
+		return fmt.Errorf("error creating request: %s", err.Error())
+	}
+
+	// Create a new HTTP client and execute the request
+	client := &http.Client{}
+	resp, err := client.Do(req)
+	if err != nil {
+		return fmt.Errorf("error fetching the latest release: %s", err.Error())
+	}
+	defer resp.Body.Close()
+
+	// Release structure to hold the data of the latest release
+	type Release struct {
+		TagName string `json:"tag_name"`
+		Name    string `json:"name"`
+		HTMLURL string `json:"html_url"`
+	}
+
+	// Decode the JSON response into the Release struct
+	var release Release
+	if err := json.NewDecoder(resp.Body).Decode(&release); err != nil {
+		return fmt.Errorf("error decoding the release data: %s", err.Error())
+	}
+
+	if AppVersion != "" {
+		v1, err := version.NewVersion(AppVersion)
+		if err != nil {
+			return fmt.Errorf("error parsing version 1: %s", err.Error())
+		}
+
+		v2, err := version.NewVersion(release.TagName)
+		if err != nil {
+			return fmt.Errorf("error parsing version 2: %s", err.Error())
+		}
+
+		if v1.LessThan(v2) {
+			colorReset := "\033[0m"
+			colorYellow := "\033[33m"
+
+			updateMessage := fmt.Sprintf("New update available! "+
+				"Version %s is now ready to download and install.", v2.String())
+			fmt.Printf("%s%s%s\n", colorYellow, updateMessage, colorReset)
+
+			commandMessage := "Please run 'curl https://install.world.dev/cli! | bash' " +
+				"to install the latest version and enjoy new features and improvements."
+			fmt.Printf("%s%s%s\n\n", colorYellow, commandMessage, colorReset)
```

Comment:
Did you consider displaying this help message and then prompting the user with a message like "Do you want to install the latest version now (y/n):".

Reply:
This could be a good thing, we can indirectly force user to update the client. but it will be annoying if user didn't want update the client.

but because i think world cli need to be always updated, i'll take this approach.

Reply:
I don't think we should this imo, someone might be running on a old version of cardinal/nakama and they need to use the old version explicitly.